src/main/java/hudson/plugins/performance/reports/PerformanceReport.java

```diff
@@ -176,6 +201,10 @@ public void addSample(TaurusFinalStats sample, boolean isSummaryReport) {
                             + "name properly for each http sample: skipping sample");
             return;
         }
+        if(!isIncluded(uri)) {
```

Comment:
code style

Reply:
can you point me to codestyle rules  ?Thanks

Reply:
you can use this rules https://wiki.apache.org/jmeter/CodeStyleGuidelines